org.eclipse.mylyn-target/mylyn-e4.26.target

```diff
@@ -64,6 +64,36 @@
 			<unit id="org.mockito.mockito-core" version="0.0.0"/>
 			<unit id="org.objenesis" version="0.0.0"/>
 		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
```

Comment:
We already have all these units in reactor for merged repo. Why do we add circular dependency again?

Reply:
> We already have all these units in reactor for merged repo. Why do we add circular dependency again?

With out the added location my local build was fine but the Jenkins Build 
https://ci.eclipse.org/mylyn/job/github-mylyn/job/main/2/ has the error 
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: org.eclipse.mylyn.tests 3.26.0.qualifier
[ERROR]   Missing requirement: org.eclipse.mylyn.tests 3.26.0.qualifier requires 'osgi.bundle; org.eclipse.mylyn.bugzilla.tests 3.26.0' but it could not be found

that I could fix with the lines.

The Question is what is different between my local build and the Jenkins build?

Reply:
Now I had a more closer look. You are right, we still need these circular dependencies for some reason, I'll work to eliminate them later.